04week/loop.js

```diff
@@ -0,0 +1,61 @@
+// Array for carsInReverse
+const carsInReverse = ['Toyota', 'Ford', 'Honda', 'Chevrolet', 'Hyundai'];
+
+// For loop to list elements in carsInReverse
+console.log('List of carsInReverse elements.');
+for(let i = 0; i < carsInReverse.length; i++) {
+  console.log(carsInReverse[i] + ' ');
+}
+
+// Created persons object
+const persons = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+}
+
+// For loop to view keys
+console.log('\nKeys for persons object : ');
+for(const person in persons) {
+    console.log(`${person}`);
+}
+console.log('\n');
+//
```

Comment:
missing second part of this question: 

Then use a for...in loop and if state to console.log the value associated with the key birthDate.